docs/ipr/optional_analyses/expression_correlation.md

```diff
@@ -12,6 +12,8 @@ This plot represents the pairwise correlation of the RNA expression of the curre
 
 ![expression correlation plot](../images/expression_correlation.png)
 
+An example of how this type of plot is created can be found in the scripting examples [here](../../scripting/RNA_Expression_Correlation).
```

Comment:
i get a 404 when i click on this link, locally

Reply:
Did you build the docs and then serve them? the dev server does some strange things sometimes with routing

```bash
mkdocs build
python -m http.server <build dir>
```

Reply:
i followed the readme instructions ending with mkdocs serve. i'll try this instead

Reply:
just fyi when i run mkdocs build i get the warning " WARNING -  Redirect target 'graphkb/scripting/Intro_Tutorial.pynb' does not exist! "

Reply:
I still get the 404 when I run it that way (Error response
Error code: 404

Message: File not found.

Error code explanation: HTTPStatus.NOT_FOUND - Nothing matches the given URI.)

Reply:
ok odd. I will try that out again on my end. I know there were some differences in the dev vs prod routing i was having trouble with

Reply:
This was my bad. I forgot this got renamed to expression similarity not expression correlation (b/c I added the PCA plots). It is fixed now.